problems/getCountriesSortedByPopulation.js

```diff
@@ -10,8 +10,11 @@
 * 
 */
 
-function getCountriesSortedByPopulation() {
-
+function getCountriesSortedByPopulation(arr) {
+    let newArr = []
+    let sorted = arr.sort((a, b) => b.population - a.population)
+    sorted.forEach(e => newArr.push(e.country))
```

Comment:
Since .map() returns an array, can you write this function so that you don't need newArr?